src/workerd/io/actor-cache.h

```diff
@@ -451,7 +451,32 @@ class ActorCache final: public ActorCacheInterface {
   const SharedLru& lru;
   OutputGate& gate;
 
-  kj::List<Entry, &Entry::link> dirtyList;
+  class DirtyList {
+    // Wrapper around kj::List that keeps track of the total size of all elements.
+  public:
+    void add(Entry& entry) {
+      inner.add(entry);
+      innerSize += entry.size();
+    }
+
+    void remove(Entry& entry) {
+      inner.remove(entry);
+      innerSize -= entry.size();
+    }
+
+    size_t size() {
```

Comment:
Can we please give this method a different name? `size()` is such an idiomatic name for accessing the length of the list that I worry it could be misleading to a future caller who wants to know how many entries are in the list.